osmaxx/excerptexport/models/export.py

```diff
@@ -80,11 +80,17 @@ def status_update_url(self):
 
     def set_and_handle_new_status(self, new_status, *, incoming_request):
         assert new_status in dict(self.STATUS_CHOICES)
+        if self.status == new_status and self.last_update_overdue():
+            new_status = self.FAILED
```

Comment:
If `set_and_handle_new_status` has been called from `osmaxx.job_progress.views.tracker()`, this doesn't make a lot of sense. So this logic should probably be moved to `osmaxx.job_progress.middleware.update_export()`.